ephios/api/templates/oauth2_provider/authorize.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+    {% if not error %}
+            <form id="authorizationForm" method="post">
+                <h3 class="block-center-heading">{% trans "Authorize" %} {{ application.name }}?</h3>
+                {% csrf_token %}
+
+                {% for field in form %}
+                    {% if field.is_hidden %}
+                        {{ field }}
+                    {% endif %}
```

Comment:
Havn't looked at the authorize page yet but why are we rendering hidden fields here?

Reply:
they explain that in the documentation, they rely on that to carry the information through from the initial request to the validation of the form the user submits

Reply:
can you add a comment pls?